sys/net/gnrc/network_layer/ipv6/nib/_nib-internal.c

```diff
@@ -226,6 +232,38 @@ void _nib_nc_remove(_nib_onl_entry_t *node)
     _nib_onl_clear(node);
 }
 
+static inline void _get_l2addr_from_ipv6(uint8_t *l2addr,
+                                         const ipv6_addr_t *ipv6)
+{
+    memcpy(l2addr, &ipv6->u64[1], sizeof(uint64_t));
+    l2addr[0] ^= 0x02;
```

Comment:
Remember: The semantic of the unique/global bit (second LSB of the most significant byte) are switched in EUI-64 and IPv6 IID. This is not a matter of the bit being set or cleared, but being set or cleared *iff* it isn't set or cleared in the IID.